cluster/manifests/deletions.yaml

```diff
@@ -5,13 +5,14 @@
 #   kind: deployment
 
 # everything defined under here will be deleted before applying the manifests
-pre_apply:
-- name: logging-agent
-  namespace: visibility
-  kind: daemonset
-- name: logging-agent
-  namespace: visibility
-  kind: secret
+pre_apply: []
```

Comment:
The `kube-dns` service needs to be deleted here in order to overwrite it with the new `coredns` service which uses the same ip.

Reply:
@mikkeloscar would create a downtime